e2e/cypress/integration/markdown/markdown_image_spec.js

```diff
@@ -17,8 +17,8 @@ describe('Markdown', () => {
 
     const baseUrl = Cypress.config('baseUrl');
 
-    const inlineImage1 = `<h3 class="markdown__heading">In-line Images</h3><p>Mattermost/platform build status:  <a class="theme markdown__link" href="https://travis-ci.org/mattermost/platform" rel="noreferrer" target="_blank"><img alt="Build Status" class="markdown-inline-img" src="${baseUrl}/api/v4/image?url=https%3A%2F%2Ftravis-ci.org%2Fmattermost%2Fplatform.svg%3Fbranch%3Dmaster"></a></p>`;
-    const inlineImage2 = `<h3 class="markdown__heading">In-line Images</h3><p>GitHub favicon:  <img alt="Github" class="markdown-inline-img" src="${baseUrl}/api/v4/image?url=https%3A%2F%2Fgithub.githubassets.com%2Ffavicon.ico"></p>`;
+    const inlineImage1 = `<h3 class="markdown__heading">In-line Images</h3><p>Mattermost/platform build status:  <a class="theme markdown__link" href="https://travis-ci.org/mattermost/platform" rel="noreferrer" target="_blank"><button class="style--none" alt="Build Status" aria-label="file thumbnail"><img alt="image placeholder" src="${baseUrl}/api/v4/image?url=https%3A%2F%2Ftravis-ci.org%2Fmattermost%2Fplatform.svg%3Fbranch%3Dmaster"></button></a></p>`;
```

Comment:
HTML strcuture change makes sense but The missing `markdown-inline-` class is actually important. My guess is that master has an issue with this now and AM is fixing it. Let's wait for that PR and run this again before going ahead with this.

Reply:
Yup I agree.  I've seen the thread on it, thanks

Reply:
Now updated per latest change